test/validation/api/ipsec/ipsec.c

```diff
@@ -16,11 +16,25 @@
 #include "test_vectors.h"
 #include "reass_test_vectors.h"
 
+#define MAX_EVENT_CACHE 3
```

Comment:
EVENT_CACHE_SIZE or MAX_CACHED_EVENTS would be easier to understand.

Reply:
Ack. Renamed to EVENT_CACHE_SIZE.